src/favicon.c

```diff
@@ -172,7 +172,7 @@ favicon_get_urls (subscriptionPtr subscription, const gchar *html_url)
 	}
 
 	/* case 2: */
-	if (html_url) {
+	if (html_url && strstr (html_url, "://")) {
 		tmp = g_strstrip (g_strdup (html_url));
```

Comment:
It might be better to use Glib's String utility functions where possible.

Reply:
Good point. Changed.